tests/test_core.py

```diff
@@ -7179,6 +7179,12 @@ def encode_utf8(data):
       else:
         return data
 
+    def source_map_file_loc(name):
+      if shared.Settings.WASM_BACKEND:
+        return name
+      # in fastcomp, we have the absolute path, which is not good
```

Comment:
I don't really understand this comment - can you please clarify why/how fastcomp is special?

Reply:
Oh, fastcomp just uses a different code path for source map generation. We are deprecating and planning to remove fastcomp, so I don't think it's worth doing double work to fix that code path too.

Reply:
Ah ok. But I think unconditionally returning `name` should still just work in both modes? (`os.path.relpath` seems to perform `abspath` internally anyway, so it doesn't require an absolute path as an input)

Reply:
Ah, this is test, not implementation, so it's output paths that are different. Seems fine then :)